src/ExtensionsSample/ErrorNotifier.cs

```diff
@@ -71,15 +71,17 @@ public void WebNotify(TraceFilter filter)
         /// <param name="filter">The <see cref="TraceFilter"/> that triggered the notification.</param>
         public void EmailNotify(TraceFilter filter)
         {
-            SendGridMessage message = new SendGridMessage()
+            Mail message = new Mail()
             {
                 From = _sendGridConfig.FromAddress,
-                Subject = "WebJob Error Notification",
-                Text = filter.GetDetailedMessage(5)
+                Subject = "WebJob Error Notification"
             };
-            message.To = new MailAddress[] { _sendGridConfig.ToAddress };
+            message.AddContent(new Content("text/plain", filter.GetDetailedMessage(5)));
+            Personalization personalization = new Personalization();
+            personalization.AddTo(_sendGridConfig.ToAddress);
+            message.AddPersonalization(personalization);
 
-            _sendGrid.DeliverAsync(message);
+            _sendGrid.client.mail.send.post(requestBody: message.Get());
```

Comment:
SendMessageAsync is our method on the client wrapper ISendGridClient (for mocking). This is the actual SendGrid API for sending the request. This sample has always used the SendGrid API directly (it's showing how to use a `TraceFilter`) so I kept it that way.